src/Integration/Laravel/Filesystem/composer.json

```diff
@@ -16,7 +16,7 @@
         "async-aws/flysystem-s3": "^1.0",
         "async-aws/simple-s3": "^1.0",
         "illuminate/filesystem": "^6.18.13 || ^7.10 || ^8.0",
```

Comment:
Looks like `illuminate/filesystem 9` works only with FlySystem 2.x (`FilesystemManager::createFlysystem` expect a `League\Flysystem\FilesystemAdapter` introduced in 2.0)

Reply:
well... `illuminate/queue: 9` requires `illuminate/filesystem: 9` 
+ version `8.0` require `php ^7.x` which prevent the `root dev` test suite to test this version